src/RuleTicket.php

```diff
@@ -858,6 +895,13 @@ public function getActions()
         $actions['assign_appliance']['force_actions']         = ['assign','regex_result', 'append'];
         $actions['assign_appliance']['appendto']              = 'items_id';
 
+        $actions['assign_project']['name']                  = _n('Associated element', 'Associated elements', Session::getPluralNumber()) . " : " . Project::getTypeName(1);
+        $actions['assign_project']['type']                  = 'dropdown';
+        $actions['assign_project']['table']                 = 'glpi_projects';
+        $actions['assign_project']['permitseveral']         = ['append'];
+        $actions['assign_project']['force_actions']         = ['assign','regex_result', 'append'];
+        $actions['assign_project']['appendto']              = 'items_id';
```

Comment:
Not sure it is used.
```suggestion
        $actions['assign_project']['appendto']              = '_projects_id';
```

Reply:
fixed